straxen/plugins/events/event_area_per_channel.py

```diff
@@ -22,26 +22,22 @@ class EventAreaPerChannel(strax.LoopPlugin):
 
     def infer_dtype(self):
         # setting data type from peak dtype
-        pfields_=self.deps['peaks'].dtype_for('peaks').fields
-        ## Populating data type
-        infoline = {'s1': 'main S1', 
-                    's2': 'main S2', 
+        pfields_ = self.deps['peaks'].dtype_for('peaks').fields
+        # populating data type
+        infoline = {'s1': 'main S1',
+                    's2': 'main S2',
                     'alt_s1': 'alternative S1',
                     'alt_s2': 'alternative S2',
                    }
         dtype = []
-        # populating APC and waveform samples
+        # populating APC
         ptypes = ['s1', 's2', 'alt_s1', 'alt_s2']
         for type_ in ptypes:
-            dtype +=[((f'Area per channel for {infoline[type_]}', f'{type_}_area_per_channel'),
+            dtype += [((f'Area per channel for {infoline[type_]}', f'{type_}_area_per_channel'),
                      pfields_['area_per_channel'][0])]
-            dtype +=[((f'Waveform for {infoline[type_]} [ PE / sample ]', f'{type_}_data'),
-                     pfields_['data'][0])]
-            dtype +=[((f'Top waveform for {infoline[type_]} [ PE / sample ]', f'{type_}_data_top'),
-                     pfields_['data_top'][0])]
-            dtype +=[((f'Length of the interval in samples for {infoline[type_]}', f'{type_}_length'),
+            dtype += [((f'Length of the interval in samples for {infoline[type_]}', f'{type_}_length'),
                      pfields_['length'][0])]
-            dtype +=[((f'Width of one sample for {infoline[type_]} [ns]', f'{type_}_dt'),
+            dtype += [((f'Width of one sample for {infoline[type_]} [ns]', f'{type_}_dt'),
```

Comment:
I keep it for now because I am not sure if anyone will be interested in the time scale of the hit patterns of S1s/S2s, as they may also provide some information on top of the hit pattern.

Reply:
`*_dt` are required in `event_top_bottom_params` https://github.com/XENONnT/straxen/blob/40f75b192a5dfbe81a2220e168086eb307cc1a8b/straxen/plugins/events/event_top_bottom_params.py#L70.

Reply:
That is actually a good catch, but in a different regard. Should not `EventTopBottomParams` https://github.com/XENONnT/straxen/blob/40f75b192a5dfbe81a2220e168086eb307cc1a8b/straxen/plugins/events/event_top_bottom_params.py#L13 depend now on `event_waveform` instead of `event_area_per_channel`? The timing properties are certainly computed based on the peak waveforms. Thus you only need the `dt` field when providing the peak waveforms and not the hit patterns.

Reply:
Can you check if there are any other dependencies on `event_area_per_channel`? If time permits we should also redo: https://straxen.readthedocs.io/en/latest/reference/datastructure_nT.html

Reply:
Hi Daniel, I just checked the dependencies and processed to `event_info`. Seems fine to me.